CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # IdentityCache changelog
 
+#### 0.6.0 Unreleased
```

Comment:
Since it was already the default and deprecated, it might only require a minor? 

I really don't care about administration :+1: